packages/backend/src/apps/m365-excel/__tests__/common/sanitise-input.test.ts

```diff
@@ -0,0 +1,56 @@
+import { describe, expect, it } from 'vitest'
+
+import { sanitiseFormulaInput } from '../../common/sanitise-input'
+
+describe('sanitise input for excel variables', () => {
+  describe('no excel formula in input', () => {
+    it('no excel formula in input', () => {
+      expect(sanitiseFormulaInput('variable1')).toEqual('variable1')
+    })
+
+    it('allowed formula in input', () => {
+      expect(sanitiseFormulaInput('=SUM(1,2)')).toEqual('=SUM(1,2)')
+    })
+  })
+
+  describe('one excel formula in input', () => {
+    it('case sensitive check in input', () => {
+      expect(sanitiseFormulaInput('hyperlink test')).toEqual(`'hyperlink test`)
+    })
+
+    it('start of input', () => {
+      expect(
+        sanitiseFormulaInput('=HYPERLINK("https://google.com", "test link")'),
+      ).toEqual(`='HYPERLINK("https://google.com", "test link")`)
```

Comment:
Just to illustrate the point, this is the outcome that we want to achieve:

```suggestion
      ).toEqual(`'=HYPERLINK("https://google.com", "test link")`)
```

Reply:
ok yea initially i didn't do this because a variable once sanitised could be anywhere in the input (the variable can don't start with the apostrophe) so lemme figure out a way to detect that the input has been sanitised